src/rfkill/rfkill.c

```diff
@@ -257,35 +268,93 @@ static int load_state(
         return 0;
 }
 
-static int save_state(
+static void save_state_queue_remove(int idx, char *state_file) {
+
+        struct write_queue_item *item, *tmp;
+
+        LIST_FOREACH_SAFE(queue, item, tmp, write_queue) {
+                if ((state_file && strcmp(item->file, state_file) == 0) || idx == item->rfkill_idx) {
```

Comment:
nitpick: we prefer streq() over strcmp() == 0 in systemd sources, see CODING_STYLE.